mtags-interfaces/src/main/java/scala/meta/pc/PresentationCompiler.java

```diff
@@ -157,6 +157,11 @@ public abstract CompletableFuture<List<AutoImportsResult>> autoImports(String na
 	 */
 	public abstract CompletableFuture<List<TextEdit>> insertInferredType(OffsetParams params);
 
+	/**
+	 * Return the missing method
+	 */
+	public abstract CompletableFuture<List<TextEdit>> insertInferredMethod(OffsetParams params);
```

Comment:
Could we reuse the codeAction interface, it should be universal for all actions

Reply:
is there an example by any chance? Otherwise I will pick this up at the end, after making sure the cases I handled in the tests work running interactively (not all do yet!)

Reply:
I was supposed to quickly create an example, but then figured we can refactor the old methods in https://github.com/scalameta/metals/pull/6881

It's more complicated for the older action, but for anything new(as soon as my PR is merged) it should be as easy as:
1. Add a new CodeActionId string
2. Implement maybeCodeActionId from CodeAction trait
3. Add handling of new id to codeAction method
4. Add id to list of supported ones
5. Use codeAction method from compilers in handleCommand

Feel free to work on the meat of the issue in the compiler, we can get back to it later or I can fix that myself.

Reply:
right, I seem to have achieved what I wanted! Things are working now :sun_with_face: